core/kits/manager.php

```diff
@@ -151,6 +151,55 @@ public function get_kit_for_frontend() {
 		return $kit;
 	}
 
+	/**
+	 * Map Scheme To Global
+	 *
+	 * Convert a given scheme value to its corresponding default global value
+	 *
+	 * @param string $type 'color'/'typography'
+	 * @param $value
+	 * @return mixed
+	 */
+	private function map_scheme_to_global( $type, $value ) {
+		$schemes_to_globals_map = [
+			'color' => [
+				'1' => Colors_And_Typography::COLOR_PRIMARY,
+				'2' => Colors_And_Typography::COLOR_SECONDARY,
+				'3' => Colors_And_Typography::COLOR_TEXT,
+				'4' => Colors_And_Typography::COLOR_ACCENT,
+			],
+			'typography' => [
+				'1' => Colors_And_Typography::TYPOGRAPHY_PRIMARY,
+				'2' => Colors_And_Typography::TYPOGRAPHY_SECONDARY,
+				'3' => Colors_And_Typography::TYPOGRAPHY_TEXT,
+				'4' => Colors_And_Typography::TYPOGRAPHY_ACCENT,
+			],
+		];
+
+		return $schemes_to_globals_map[ $type ][ $value ];
+	}
+
+	/**
+	 * Convert Scheme to Default Global
+	 *
+	 * If a control has a scheme property, convert it to a default Global.
+	 *
+	 * @param $scheme - Control scheme property
+	 * @return array - Control/group control args
+	 * @since 3.0.0
+	 * @access public
+	 *
+	 */
+	public function convert_scheme_to_global( $scheme ) {
+		if ( isset( $scheme['type'] ) && isset( $scheme['value'] ) ) {
+			//_deprecated_argument( $args['scheme'], '3.0.0', 'Schemes are now deprecated - use $args[\'global\'] instead.' );
```

Comment:
Is it added to the deprecation sheet?

Reply:
There's no template for PHP Vars in the deprecation sheet. Should I add one?

Reply:
Yes, add one.

Reply:
Done